### ISSUE TITLE
Continous Delivery (Second Edition German)

### ISSUE THREAD

**Issue by raynor84** [ewolff/user-registration-V2#11]
Hi,

i'm trying to get through your book. I'm curently on page 44. Could it be that chef-solo and ubuntu have newer versions. I wasn
't able to get the code running. The older version (ubuntu 16.04 doesn't have updates and so programs are not installable).

Kind regards

Stefan

**Comment by ewolff** [comment]
This might very well be the case and sorry about not providing updates. Did you try to run it on a more recent version of Ubuntu? I guess that should work, but I am not entirely sure.

**Comment by raynor84** [comment]
yes currently i have a problem with the new tomcat version I tried to change it. But I didn't get it to work yet. I also thought about doing a complete init of chef-solo. But I would need to understand the concept further.

**Comment by raynor84** [comment]
might there be an update of the code?
